### ISSUE TITLE
Support different CleanWebpackPlugin versions

### ISSUE THREAD

**Issue by iki** [sysgears/larix#7]
API changed with [CleanWebpackPlugin v2 and v3](https://github.com/johnagan/clean-webpack-plugin/releases/).

Zen afaik doesn't detect versions and [uses only v1 API](https://github.com/sysgears/larix/blob/master/packages/zen/src/plugins/WebpackPlugin.ts#L73).

Using Zen 0.1.30 with CleanWebpackPlugin 2.0.2 throws:
```
zen error Error: clean-webpack-plugin only accepts an options object. See: https://github.com/johnagan/clean-webpack-plugin#options-and-defaults-optional
    at new CleanWebpackPlugin (.\my-project\node_modules\clean-webpack-plugin\src\clean-webpack-plugin.ts:79:19)
    at createPlugins (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:74:30)
    at createConfig (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:294:19)
    at WebpackPlugin.configure (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:552:50)
    at .\my-project\node_modules\@larix\zen\src\createBuilders.ts:146:62
    at Array.forEach (<anonymous>)
    at _loop_2 (.\my-project\node_modules\@larix\zen\src\createBuilders.ts:146:21)
    at Object.createBuilders [as default] (.\my-project\node_modules\@larix\zen\lib\createBuilders.js:154:9)
    at Object.<anonymous> (.\my-project\node_modules\@larix\zen\src\cli.ts:50:30)
    at Module._compile (internal/modules/cjs/loader.js:777:30)
```

Using Zen 0.1.30 with CleanWebpackPlugin 3.0.0 throws:
```
zen error TypeError: CleanWebpackPlugin is not a constructor
    at createPlugins (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:74:30)
    at createConfig (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:294:19)
    at WebpackPlugin.configure (.\my-project\node_modules\@larix\zen\src\plugins\WebpackPlugin.ts:552:50)
    at .\my-project\node_modules\@larix\zen\src\createBuilders.ts:146:62
    at Array.forEach (<anonymous>)
    at _loop_2 (.\my-project\node_modules\@larix\zen\src\createBuilders.ts:146:21)
    at Object.createBuilders [as default] (.\my-project\node_modules\@larix\zen\lib\createBuilders.js:154:9)
    at Object.<anonymous> (.\my-project\node_modules\@larix\zen\src\cli.ts:50:30)
    at Module._compile (internal/modules/cjs/loader.js:777:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:788:10)
```

**Comment by vlasenko** [comment]
@iki Thank you for the issue report. Would you like to submit the pull request to fix this issue?

**Comment by larixer** [comment]
Fixed in #23 `@larix/zen@0.1.36`
